services/state/roster/roster_state.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package com.hedera.hapi.node.state.roster;
+
+/*
+ * Hedera Network Services Protobuf
+ *
```

Comment:
These two lines should not be present.
Also, check for hidden zero-width characters if you copied this from an old proto file.